hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -132,9 +132,11 @@ public void init() throws DBException {
     }
 
     try {
-      THREAD_COUNT.getAndIncrement();
       synchronized(THREAD_COUNT) {
-        connection = ConnectionFactory.createConnection(config);
+        THREAD_COUNT.getAndIncrement();
```

Comment:
this change makes it so that all of our accesses for THEAD_COUNT are now in synchronized blocks, so it need not be an AtomicLong.

Would you mind changing it to a normal Long with a comment that it must be accessed within a synchronized block?